### ISSUE TITLE
Need page breaks or margins during printing

### ISSUE THREAD

**Issue by jameswhite4684** [digitalocean/netbox#3245]
### Environment
* Python version:  
* NetBox version:  2.5.13

### Proposed Functionality

Printing needs to be enhanced / fixed.
Tested in Chrome and IE

Issue #1:
Page breaks or margins are needed during printing. (Works ok in IE not in Chrome)

Issue #2:
Print font size seems to be increased dramatically, reducing size may help table printing from distorting.

Issue #3:
Rack Group view printing displays scrollbar and white space where search bar would be, thus distorting the print job.

### Use Case

To print and document for auditing and disaster recover and easily being read.

**Comment by DanSheps** [comment]
Is this not mostly covered by #160?

What else would you be looking to print out?  All the data contained within devices?  I think this could be unwieldy and cumbersome.

There could be instances where you would have thousands of devices in netbox, which could result in multiple binders if you are planning on that route.

**Comment by jameswhite4684** [comment]
So, I looked through those and could not find it directly related or was closed. Issue #1 and #2 can probably be removed as Chrome does have a scaling ability which helps greatly and figured out the additional setting to get the margins corrected.

I really think Issue #3 does need to be fixed. I have attached a pdf, if it is able to be attached to show what I mean.

[Rack Elevations - NetBox.pdf](https://github.com/digitalocean/netbox/files/3258906/Rack.Elevations.-.NetBox.pdf)
